react/features/conference/components/web/ToggleTopPanelLabel.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+// @ts-ignore
+import { IconMenuDown } from '../../../base/icons';
+// @ts-ignore
+import { Label } from '../../../base/label';
+// @ts-ignore
+import { Tooltip } from '../../../base/tooltip';
+// @ts-ignore
+import { setTopPanelVisible } from '../../../filmstrip/actions.web';
+
+const ToggleTopPanelLabel = () => {
+    const asd = 'hello';
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
+    const topPanelVisible = useSelector((state: any) => state['features/filmstrip'].topPanelVisible);
+    const onClick = useCallback(() => {
+        dispatch(setTopPanelVisible(true));
+    }, []);
+
+    return !topPanelVisible && (<Tooltip
```

Comment:
kinda weird to use it like this, maybe rename to topPanelHidden and negate the selector result?